.github/actions/setupconda/action.yml

```diff
@@ -19,6 +19,6 @@ runs:
       with:
         miniforge-version: latest
         auto-update-conda: false
-        environment-file: conda/conda-${{ runner.os }}-64.lock
+        environment-file: conda/conda-${{ runner.os == 'Linux' && 'linux' || 'osx'  }}-64.lock
```

Comment:
@jomey you are a bash wiz!

Reply:
Credit goes to the comment on the open issue for GH actions
https://github.com/actions/runner/issues/409